README.md

```diff
@@ -1,5 +1,6 @@
 # Sneakers
 
+[![Build Status](https://travis-ci.org/jondot/sneakers.svg?branch=master)](https://travis-ci.org/jondot/sneakers)
```

Comment:
@pbollenbeck Just in case is was unintentional, this is the second badge added to the README (the first is on line 23) 😄.